tidb-lightning/tidb-lightning-physical-import-mode-usage.md

```diff
@@ -221,3 +221,26 @@ If the table is large, Lightning will split the table into multiple batches of 1
 After the file data is read, Lightning needs to do some post-processing, such as encoding and sorting the data locally. The concurrency of these operations is controlled by `region-concurrency`. The default value is the number of CPU cores. You can leave this configuration in the default value. It is recommended to deploy Lightning on a separate server from other components. If you must deploy Lightning together with other components, you need to lower the value of `region-concurrency` according to the load.
 
 The [`num-threads`](/tikv-configuration-file.md#num-threads) configuration of TiKV can also affect the performance. For new clusters, it is recommended to set `num-threads` to the number of CPU cores.
+
+## Configure disk quota <span class="version-mark">New in v6.2.0</span>
+
+When you imports data in physical import mode, TiDB Lightning creates a large number of temporary files on the local disk. These files are used to encode, sort, and split the original data. When the local disk space is insufficient, TiDB Lightning reports an error and exits because of write failure.
```

Comment:
```suggestion
When you import data in physical import mode, TiDB Lightning creates a large number of temporary files on the local disk to encode, sort, and split the original data. When the local disk space is insufficient, TiDB Lightning reports an error and exits because of write failure.
```